server/pulp/plugins/util/publish_step.py

```diff
@@ -1046,36 +1069,36 @@ def cancel(self):
         self.downloader.cancel()
 
 
-class GetLocalUnitsStep(PluginStep):
+class SaveUnitsStep(PluginStep):
+    """
+    Any step that saves units to the database should use this step in order to ensure that
```

Comment:
Consider putting a 1 sentence short description as the first part of the docstring, then a newline, then this longer description.